server.js

```diff
@@ -1,18 +1,41 @@
 'use strict';
 const express = require('express');
+const request = require('request');
 
 const config = require('./config');
 const log = require('./logger');
 
 const app = express();
-const port = config.SERVER_PORT;
+
+const PORT = config.SERVER_PORT;
+const API_ROOT = config.API_ROOT || 'http://localhost:8081/';
 
 app.get('/healthz', function healthEndpoint(req, res) {
     res.send({ env: config.ENV, status: 'OK' });
 });
 
-const server = app.listen(port, () => {
-    log.info('ui on:' + port);
+app.get('/', function forwardRequestForStatistics(req, res) {
+    const requestOptions = {
+        url: '/statistics',
+        baseUrl: API_ROOT,
+        qs: req.query,
+        headers: {
+            'x-auth-username': req.get('x-auth-username'),
+            'x-auth-userid': req.get('x-auth-userid')
+        }
+    };
+    request(requestOptions, (error, response, body) => {
+        if (error) {
+            log.error(`Error: ${error}`);
+        }
+        res.status(response.statusCode)
```

Comment:
I'm not keen on the naming of "res" and "response".  It's not really clear which is which or why there are two.